app/models/osf-model.ts

```diff
@@ -187,20 +186,15 @@ export default class OsfModel extends Model {
         relatedModel: OsfModel,
     ) {
         const apiRelationshipName = underscore(relationshipName);
-        let url = getSelfHref(this.relationshipLinks[apiRelationshipName]);
+        const url = getSelfHref(this.relationshipLinks[apiRelationshipName]);
```

Comment:
Looks like this should wait until #615 is merged, is that right?

Reply:
Correct.